units/systemd-binfmt.path

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=Path activation for additional binary formats
+Documentation=man:systemd-binfmt.service(8) man:binfmt.d(5)
+Documentation=https://www.kernel.org/doc/Documentation/binfmt_misc.txt
+
+[Path]
+DirectoryNotEmpty=/lib/binfmt.d
+DirectoryNotEmpty=/usr/lib/binfmt.d
+DirectoryNotEmpty=/usr/local/lib/binfmt.d
```

Comment:
I just took them from the [systemd-binfmt.service](https://github.com/systemd/systemd/blob/master/units/systemd-binfmt.service.in#L17-L21) unit.

I thought if the service looked at those paths, then it made sense to have the path unit look at the same ones.